OpenVBX/views/messages/details.php

```diff
@@ -124,6 +124,43 @@
 			<h3>Transcription</h3>
 			<?php endif; ?>
 			<div class="message-transcript"><?php echo (is_null($message->content_text) ? "(no transcription)" : $message->content_text) ?></div>
+			<?php if(substr($message->call_sid,0,2) == 'MM'):
+				echo "<br><br><h3>Media Attachements</h3>";
+				
+				$ci = & get_instance();
+				
+				require_once(APPPATH . 'libraries/twilio.php');
```

Comment:
Also, let me know if the Twilio library gives you any crap here. The included library is a bit out of date and may be missing some fixes. If it is out of date either you can update it in your branch or I can update it in this repo and you can pull from upstream to get the changes.